app/models/term.py

```diff
@@ -14,3 +15,34 @@ class Term(baseModel):
     isAcademicYear          = BooleanField(default=False)
     isFinalEvaluationOpen   = BooleanField(default=False)
     isMidyearEvaluationOpen = BooleanField(default=False)
+
+
+    @staticmethod
+    def order_by_term(queryResult, *, reverse=False):
+        """
+        Accepts the results of a query where each object has a `termCode` attribute.
+        To bring selected columns from other tables into an objects direct attributes
+        use the .objects() method on the query.
+
+        Sorts by the Term Code in logical order based first on year and then by the seasonalCode
+        
+        seasonalCode := last two digits of the term code which maps arbitrarily to the name of the term, break, etc.
+        """
+        seasonalCodeToOrderValue = defaultdict(lambda: 1)
+        seasonalCodeToOrderValue.update({
+            '00' : 0,
+            '11' : 2,
+            '04' : 3,
+            '01' : 4,
+            '02' : 5,
+            '12' : 6,
+            '05' : 7,
+            '03' : 8,
+            '13' : 9,
+
```

Comment:
This dictionary maps the last two digits of the term code to numbers. I wonder why you skipped 1 in that case.

Reply:
We skipped 1 because all terms that have never been identified before should default to 1 as per line 31 here